pandas/core/arrays/sparse.py

```diff
@@ -1697,6 +1697,17 @@ def __array_ufunc__(self, ufunc, method, *inputs, **kwargs):
             # No alignment necessary.
             sp_values = getattr(ufunc, method)(self.sp_values, **kwargs)
             fill_value = getattr(ufunc, method)(self.fill_value, **kwargs)
+
+            if isinstance(sp_values, tuple):
```

Comment:
did u test divmod ?

Reply:
Already tested. It goes through the dispatch to `SparseArray.__divmod__`. We didn't previously have any tests for ufuncs returning multiple outputs that aren't also a dunder method.